src/components/login/loginService.ts

```diff
@@ -0,0 +1,55 @@
+import { setLoggedIn, setLoggingIn } from "./state/statusAction";
+
+import { CognitoTokens } from "interfaces/cognitoTokens";
+import { parseToken } from "./tokenParser";
+import store from "../../state/index";
+
+const getDeployedRedirectUrl = (): string => {
+    return "https://" + window.location.host + "/index.html";

```

Comment:
```suggestion
    return `https://${window.location.host}/index.html`;

```
would be more consistent with the other string construction in this class.
Also you could remove a few unnecessary return statements in this class.